core/src/main/java/io/questdb/cairo/TableReader.java

```diff
@@ -144,7 +148,7 @@ public TableReader(
                 openPartitionInfo.setQuick(baseOffset, partitionTimestamp);
                 openPartitionInfo.setQuick(baseOffset + PARTITIONS_SLOT_OFFSET_SIZE, -1L); // -1L means it is not open
                 openPartitionInfo.setQuick(baseOffset + PARTITIONS_SLOT_OFFSET_NAME_TXN, txFile.getPartitionNameTxn(i));
-                openPartitionInfo.setQuick(baseOffset + PARTITIONS_SLOT_OFFSET_COLUMN_VERSION, columnVersionReader.getMaxPartitionVersion(partitionTimestamp));
+                openPartitionInfo.setQuick(baseOffset + PARTITIONS_SLOT_OFFSET_COLUMN_VERSION, -2);
```

Comment:
what does -2 mean and why?

Reply:
It means it's not open, I can revert it here, it doesn't matter

Reply:
a comment or constant will do, don't need to revert, just making it easier to follow

Reply:
added comment in [c50ecb8](https://github.com/questdb/questdb/pull/5010/commits/c50ecb8b8e01b475dd6d675d34eb96ad62977db1)